scrapeWorks/server.js

```diff
@@ -0,0 +1,55 @@
+var request = require("request");
+var fs = require('fs');

```

Comment:
Is this script still needed? It is not getting called from /musicbrainz/musicbrainzcli/server.js